beam-postgres/examples/test/pagilatest.hs

```diff
@@ -27,7 +31,9 @@ import Data.Time.LocalTime (LocalTime)
 import Data.Conduit ((=$=), runConduit)
 import qualified Data.Conduit.List as CL (mapM_)
 
-testQuery conn q = runConduit (runSelect conn (select q) =$= CL.mapM_ (putStrLn . show))
+-- TODO
+-- runSelect deprecated?
```

Comment:
Is there a problem adjusting `testQuery` to use `streamingRunSelect`, which replaces `runSelect`?

Reply:
Done; there is a type error I can't figure out. Would you complete it?

Reply:
This is because you can't use `runConduit`; you need to use `runConduitRes`, which wraps `IO` in a `ResourceT`. Part of the module becomes:

```haskell
import Data.Conduit ((.|), runConduitRes)
import qualified Data.Conduit.List as CL (mapM_)
import Database.Beam.Postgres.Conduit (streamingRunSelect)

testQuery :: ( FromBackendRow Postgres (QExprToIdentity res)
             , Projectible Postgres res, Show (QExprToIdentity res)
             ) => Connection 
               -> Q Postgres db QBaseScope res 
               -> IO ()
testQuery conn q 
  = runConduitRes (
         streamingRunSelect conn (select q)
      -- need (liftIO . putStrLn) because we're in the ResourceT IO monad, not IO 
      .| CL.mapM_ (liftIO . putStrLn . show)
   )

```

Reply:
done, thanks, I will study this